lib/modules/datasource/galaxy/index.ts

```diff
@@ -50,13 +50,19 @@ export class GalaxyDatasource extends Datasource {
       throw this.handleGenericErrors(err);
     }
 
-    // istanbul ignore if
     if (body.results.length > 1) {
-      logger.warn(
-        { dependency: packageName },
-        `Received multiple results from ${galaxyAPIUrl}`,
-      );
-      return null;
+      const githubUsernames = body.results.map((result) => result.github_user);
```

Comment:
I would filter here and warn if the filtered result is empty. that way we only have to iterate once.